pom.xml

```diff
@@ -4,11 +4,10 @@
 	<artifactId>lemminx-parent</artifactId>
 	<version>0.16.1-SNAPSHOT</version>
 	<packaging>pom</packaging>
-	<name>Eclipse LemMinX</name>
```

Comment:
Do not remove project name

Reply:
fixed

Reply:
I think the name should directly be under `<project>`, instead of being under `<properties>`.

Reply:
my bad,sorry-( fixed